package.json

```diff
@@ -22,33 +22,33 @@
     "build": "npm run _rushRebuild",
     "buildci": "npm run _rushRebuildCi",
     "buildfast": "npm run _rushBuild",
-    "code-style": "node node_modules/@microsoft/rush/bin/rush code-style",
-    "change": "node node_modules/@microsoft/rush/bin/rush change",
-    "generate": "node node_modules/@microsoft/rush/bin/rush generate --force",
+    "code-style": "node common/scripts/install-run-rush.js code-style",
+    "change": "node common/scripts/install-run-rush.js change",
+    "generate": "node common/scripts/install-run-rush.js update --force",
```

Comment:
is update --force semantically the same as generate? 

Should there be an update script that runs update without `--force`?

Reply:
In Rush 5, the `rush generate` command was renamed to `rush update --full`.

I think this is a typo.  There isn't a `--force` option.

---
In reply to: [216525273](https://github.com/OfficeDev/office-ui-fabric-react/pull/6242#discussion_r216525273) [](ancestors = 216525273)

Reply:
Cliff and I spoke offline. We'll preserve `npm run generate` as `rush update --full` and add `npm run update` which will run `rush update`.

Reply:
Because NPM already has a command `npm update` that does something else, I would suggest to call it something like `npm run full-update` to avoid confusion.

That said, I don't really understand the point of this macro.  It's not saving any typing, and it makes Rush's command-line help less discoverable.

@cliffkoh

Reply:
@pgonzal Installing Rush CLI isn't a prereq for Fabric.. hence most of the documented steps refer to the npm run scripts and not rush cli commands.

(Personally, I belong in the camp which always uses rush cli commands and never these "macros").